gulp/config.js

```diff
@@ -63,11 +46,7 @@ module.exports = {
 
         // documentation site
         pldoc_src: pldocSrc + '/static/images',
-        pldoc_src_files: pldocSrc + '/static/images/**/*',
-
-        // example demo
-        demo_src: demoSrc + '/static/images',
-        demo_src_files: demoSrc + '/static/images/**/*'
+        pldoc_src_files: pldocSrc + '/static/images/**/*'
```

Comment:
Yep, this file ended up being used in like two places. I think it's adding more complexity than it's solving, so it's gone now!